contracts/deposit-service/AxelarDepositService.sol

```diff
@@ -2,12 +2,12 @@
 
 pragma solidity 0.8.9;
 
-import { SafeTokenTransfer, SafeNativeTransfer } from '@axelar-network/axelar-gmp-sdk-solidity/contracts/utils/SafeTransfer.sol';
 import { IERC20 } from '@axelar-network/axelar-gmp-sdk-solidity/contracts/interfaces/IERC20.sol';
+import { Upgradable } from '@axelar-network/axelar-gmp-sdk-solidity/contracts/upgradable/Upgradable.sol';
```

Comment:
Don't want to switch to the new upgradable in this release, since we have a few other things to focus on. Let's merge it a bit later. I wanna make sure there are no unexpected issues when switching and have folks be comfortable with the new ownership transfer method.